.gitignore

```diff
@@ -118,3 +118,12 @@ env/
 
 ### Configuration files ###
 .env
+
+### Vagrant ###
+.vagrant
+
+### Poetry ###
+poetry.lock
```

Comment:
Actually the lock file shouldn't be gitignored. It lets you control the versions of dependencies precisely, to avoid any surprising differences between servers.

Reply:
Fixed